xcm/src/v0/traits.rs

```diff
@@ -186,7 +186,7 @@ impl<C> ExecuteXcm<C> for () {
 ///
 /// # Example
 /// ```rust
-/// # use xcm::v0::{MultiLocation, Xcm, Junction, Error, OriginKind, SendXcm, Result};
+/// # use xcm::v2::{MultiLocation, Xcm, Junction, Junctions, Error, OriginKind, SendXcm, Result, Parent};
```

Comment:
Doesn't quite make sense to me that this uses v2 types when we're under the v0 directory...